ddl/serial_test.go

```diff
@@ -429,3 +429,17 @@ func (s *testSerialSuite) TestTableLocksEnable(c *C) {
 	tk.MustExec("lock tables t1 write")
 	checkTableLock(c, tk.Se, "test", "t1", model.TableLockNone)
 }
+
+// TestRunDDLJobPanic tests recover panic when run ddl job panic.
+func (s *testSerialSuite) TestRunDDLJobPanic(c *C) {
+	defer func() {
+		c.Assert(failpoint.Disable("github.com/pingcap/tidb/ddl/mockPanicInRunDDLJob"), IsNil)
+	}()
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustExec("drop table if exists t")
+	c.Assert(failpoint.Enable("github.com/pingcap/tidb/ddl/mockPanicInRunDDLJob", `1*return(true)`), IsNil)
```

Comment:
```suggestion
	c.Assert(failpoint.Enable("github.com/pingcap/tidb/ddl/mockPanicInRunDDLJob", `1*panic("panic test")`), IsNil)
```
and inject code with
```
failpoint.Inject("mockPanicInRunDDLJob", nil)
```
is simpler

Reply:
Great, thanks.